### DISCUSSION TITLE
1.21 minecraft服务器，有人来玩吗？

### DISCUSSION THREAD

**Discussion by QCG1081404q** [share121/inter-knot#2947] [其他]
欢迎大家加入服务器，希望它能越来越好～
![mmqrcode1724003540890](https://github.com/user-attachments/assets/07b4c413-3c68-46f1-8ba5-782df4a49721)

**Comment by Boring-Coils** [comment]
突然想笑，我自己开的服务器我自己都不玩

**Comment by QCG1081404q** [comment]
我倒是还好，我自己就在里面肝了两个月。
但是BDS1.21.20.24不支持一堆插件和模组，呃。。弱弱问一句：有什么解决办法吗？
₍˄·͈༝·͈˄*₎◞ ̑̑

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
